desktop/cmp/input/ButtonGroupInput.js

```diff
@@ -64,6 +78,23 @@ class Model extends HoistInputModel {
         const btns = this.domEl?.querySelectorAll('button') ?? [];
         return filter(btns, {disabled: false});
     }
+
+    active(value){
+        if (!this.enableMulti) return this.renderValue === value;
```

Comment:
probably best to use "internalValue" to be consistent with usage below. (although will be the same for this control.  Also destructure `internalValue` and consider ternary!